doc/architecture/decisions/0020-use-skylight-application-performance-monitoring.md

```diff
@@ -0,0 +1,24 @@
+# 20. use-skylight-application-performance-monitoring
```

Comment:
I'd like some information about what Skylight provides for us in here, and what the other options might have been.

Reply:
Added a bit more in!